src/views/PersonalSettingsPage.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+	<div class="gpoddersync_settings">
+		<SettingsSection :title="t('gpoddersync', 'Synced subscriptions')"
+			:description="t('gpoddersync', 'Podcast subscriptions that has so far been synchronized with this Nextcloud account.')">
```

Comment:
```suggestion
			:description="t('gpoddersync', 'Podcast subscriptions synchronized to this Nextcloud account so far.')">
```
A bit shorter. The 'has' would need to be a 'have' in either case since subscriptions is in plural.

Reply:
I make this "has" vs "have" mistake a lot. Do you have a good resource you can tip on why this is, more generally? Or is it always "when it's plural -> use have"?

Reply:
It's "has" if used with the third person singular only (he/she/it), otherwise always "have" (I, you, all plural forms). So since it's third person plural here (subscriptions), it's "have".

Not an expert in English grammar myself, so I don't know any online resources, but I think it's due to this annoying he/she/it "-s" rule one had to learn at school (me at last). You could search by that to find some material.

Reply:
Thank you so much! I don't remember such a rule, but I also don't remember much from my english class back in the day. I've bookmarked your comment so I can refer to it in the future :)